### DISCUSSION TITLE
AirDrop not working

### DISCUSSION THREAD

**Discussion by controlfix87** [osy86/HaC-Mini#929] [Q&A] [answered]
Hi,
I'm using HacMini 3.1.1 pre release.
I have the stock WiFi / BT card.
I want to send some files via AirDrop, I can see the Hackintosh on my iPhone, I select it but it's loading forever and writing Waiting.. 
Tried to reset the NVRAM 3 times in a row, no luck.
When trying to transfer to my MacBook Pro laptop, it works fast and easy as it should.

Can anyone help on that? Thanks! :)

**Comment by goodbest** [comment, marked as answer]
airdrop is not supported for stock intel card.

You should read the whole faq and instructions for openintelwifi repo

**Reply by controlfix87** [reply]
Right. 
do you know if the card in the guide will work on windows as well? 
I have 2 ssd’s and the apple wifi card, so is it possible with all of this?

**Reply by scottzoloft** [reply]
Get the wifi card from the guide. Mac OS will have a driver for it already. IIRC windows wont auto detect and install the driver but can be installed manually

**Comment by controlfix87** [comment]
Installed the guide supported wifi. Along side the ssd drive, works great! (Without the screw holding the ssd…)
The wifi speeds are terrible… I have 1Gbps but download speed is only 80mbps.. maybe there is some other stronger supported model?..

**Reply by controlfix87** [reply]
Well, connected LAN.

**Reply by KrisCris** [reply]
bcm943602cs
